python/xir/program.py

```diff
@@ -39,7 +39,7 @@ def get_floats(params: Params) -> Params:
                 params_with_floats.append(p)
 
     elif isinstance(params, Dict):
-        params_with_floats = dict()
+        params_with_floats = {}
```

Comment:
Is this because `{}` is faster than `dict()`? 😆

Reply:
Haha, that's right!

> *Consider using {} instead of dict()* Emitted when using dict() to create an empty dictionary instead of the literal {}. The literal is faster as it avoids an additional function call.